DiscordBot/Extensions/UserDBRepository.cs

```diff
@@ -10,22 +10,17 @@ public class ServerUser
         /// <summary> This is internal Database ID, remember to use UserID</summary>
         // ReSharper disable once InconsistentNaming
         public int ID { get; private set; }
```

Comment:
We can probably remove this if that doesn't cause issues for Insight.
It will remove confusions from people trying to use it.